src/app/api/dummy/route.ts

```diff
@@ -0,0 +1,15 @@
+import { client } from "@/db/client";
+import { NextRequest } from "next/server";
+
+export async function POST(request: NextRequest): Promise<Response> {
+  const { action, payload } = await request.json();
+  if (action === "add") {
```

Comment:
E' questo il modo giusto per "trattare" un'API in Next.js?

Reply:
Non capisco bene a cosa serva "action" 🤔🤔🤔

Reply:
Bha no, nulla! Volevo solo capire cosa arrivava dall'altra parte qualora dovessimo inviare dei parametri